lib/public/views/QcFlags/Create/qcFlagCreationComponent.js

```diff
@@ -128,7 +130,9 @@ export const qcFlagCreationComponent = (creationModel) => h(
     '.flex-column.g3.flex-grow',
     creationFormComponent(creationModel, [
         creationModel.run.match({
-            Success: (run) => qcFlagCreationForm(run, creationModel),
+            Success: (run) => run.runQuality === RunQualities.BAD
+                ? h('.alert.alert-danger', h('b', BAD_RUN_IN_ASYNC_QC_MESSAGE))
```

Comment:
why `b` ?

Reply:
because other errors are styles like that

Reply:
Ok that's a bad thing. However, why not using errorAlert ?

Reply:
error alert was designed to work with errors from API, here, it's different. I mean it can be done with errorAlert, then I 'd need to wrap the message, possibly, even use RemoteData.apply in fetchRun method

Reply:
errorAlert is just a convenient way to display error messages in a uniform way, I think we should use it. I typed the parameter but I should not have named it ApiError, simply `LoggableError` for example. You just have to wrap the `BAD...` in an object with simply title.

However as I said previously using a global message like this is for me a mistake, especially if you use it as info and error.

Reply:
done